plate_mapper/plate_mapper.py

```diff
@@ -127,19 +134,22 @@ def plate_mapper(input_f, barseq_f, output_f, names_f=None, special_f=None):
         # [ barcode sequence, linker primer sequence, primer plate #, well ID ]
         sample, property = '', []
         if plate in plates:
+            id = primer2plate[plate]
```

Comment:
This is not recommended, since id is a reserved keyword.  One option is to rename this variable to id_